src/test/java/core/basesyntax/strategy/handler/impl/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax.strategy.handler.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.strategy.handler.OperationHandler;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class BalanceOperationHandlerTest {
+    private OperationHandler handler;
+    private int oldValue;
+
+    @BeforeEach
+    void setUp() {
+        handler = new BalanceOperationHandler();
+        oldValue = 500;
+    }
+
+    @Test
+    void operate_positiveTransaction_ok() {
+        int transaction = 100;
```

Comment:
let's rename this variable or remove it and use literal directly. Fix the same issue in all places